### PR TITLE
Update Payments API Docs with new fields

### PR DESCRIPTION
**Pull request by N1ck**
Additional data has been added to the payments API: https://github.com/vend/webregister/pull/2802 . This PR updates the example to reflect the changes.

---

### CODE REVIEW

File: tutorials/payments_api/reference.md [hunk 1]
```diff
@@ -78,60 +84,60 @@ Messages are JSON objects with one required field.
 
 #### Required
 
-| Key | Type | Required | Description |
-| :---: | :---: | :---: |---|
-|`step`|string|required|The step in the transaction process to enact.|
-|`success`|boolean|optional|DEPRECATED: Whether the step is considered successful.|
+|    Key    |  Type   | Required | Description                                            |
```
Comment (N1ck) [inline tutorials/payments_api/reference.md]: The rest of the changes are just formatting.

File: tutorials/payments_api/reference.md [hunk 2]
```diff
@@ -43,31 +43,37 @@ Example response payload:
...
+                "name": "T-shirt",
                 "quantity": "1",
-                "unit_price": "17.39130",
-                "unit_tax": "2.60870",
-                "tax_id": "00000000-0002-0002-0002-000000000003"
+                "sku": "10001",
+                "unit_price": "23.47826",
+                "unit_tax": "3.52174",
+                "tax_id": "00000000-0002-0002-0002-000000000003",
+                "tax_rate": "0.15000",
```
Comment (laher) [inline tutorials/payments_api/reference.md]: is there an API doc we should update here too?

e.g. in here ... https://github.com/vend/vend-api-2.0-specification or here https://github.com/vend/vend-api-0.9-specification 

I couldn't see something which matched exactly, so I may be missing something.

File: tutorials/payments_api/reference.md (hunk 1 above)
Comment (laher) [reply tutorials/payments_api/reference.md]: _wonders: would another editor remove the spaces again?_ it does look like an improvement though

File: tutorials/payments_api/reference.md (hunk 2 above)
Comment (N1ck) [reply tutorials/payments_api/reference.md]: I don't think so, this isn't one of our traditional APIs as it's only transmitted over `postMessage` from Webregister.

**Pull request merged**